be/src/olap/rowset/beta_rowset_reader.cpp

```diff
@@ -237,4 +235,12 @@ Status BetaRowsetReader::next_block(vectorized::Block* block) {
     return Status::OK();
 }
 
+bool BetaRowsetReader::_should_push_down_value_predicates() const {
```

Comment:
should we check if there is a sequence column in tablet schema?

Reply:
not need. `not_overlapping()` means the rowset is compactioned, after compactioned the sequence column have same key only keep one row